prac_05/state_names.py

```diff
@@ -0,0 +1,18 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
```

Comment:
File has been reformated

Reply:
This line of comment has now been removed.